release.sos

```diff
@@ -12,9 +12,9 @@ bash: expand = True
 [singularity]
 parameter: config = path
 parameter: out_dir = path('container/singularity')
-bash: expand = True
+bash: expand = True, stderr = f'{config}.stderr', stdout = f'{config}.stdout'
   rm -f {out_dir}/{config:n}.sif
-  singularity build --fakeroot {out_dir}/{config:n}.sif container/singularity/{config}
+  singularity build --fakeroot {out_dir}/{config:n}.sif {config}
```

Comment:
in this case we should use `{out_dir}/{config:bn}.sif` otherwise you'll end up with the path of config file inside `out_dir`

Reply:
Also please add to gitignore to skip these files:

```
**/*.sif
**/*.stdout
**/*.stderr
```